### ISSUE TITLE
Admin Login Page needs an update

### ISSUE THREAD

**Issue by Tangol** [villagedefrance/OpenCart-Overclocked#113]
The **Admin Login Page** needs a styling update as it offers too much information to a potential hacker.
A good page style would be like **[this](http://demo.litecart.net/admin/login.php?redirect_url=%2Fadmin%2F)**
Again, something beautiful regarding this matter, **[here](https://mail.rainloop.net/)**
It's best not to offer any information about the system and the version being used.

Also, I would remove the code below from the source, as it offers information about OpenCart being used.
``

In other words, let's add some **[security through obscurity](https://en.wikipedia.org/wiki/Security_through_obscurity)**, to make life a little bit harder for hackers.😀 

Thanks!

**Comment by villagedefrance** [comment]
It is done, but I leave this issue open for a few more days for your comments, just in case I missed something or the code can be improved.

**Comment by Tangol** [comment]
Looks good Philippe, however when you click on **Forgotten Password**, it takes you to the standard OC page, that shows the version and everything.
I'm thinking that needs a style update too.

Also, **[here](https://medium.com/@AmJustSam/how-to-do-css-only-frosted-glass-effect-e2666bafab91)** I've found a nice effect for the `.content-login` box.
Just to make it look even more spectacular.

Thanks!

**Comment by Tangol** [comment]
Now it looks/works perfect.
I guess the report can be closed.

Thank you Philippe!

**Comment by villagedefrance** [comment]
Line 785 - margin: 60px only, not 75px as it goes to low on small mobile screens.

**Comment by Tangol** [comment]
To make the login box fully responsive, at line 563 replace 40px with **4%**, and 128px with **0**
```
#content-login {
	background: #191919;
	padding: 4% 10px 0 10px;
	min-width: 300px;
}
```

and at line 796 replace 60px with **5%**.
```
.box-login {
	position: relative;
	display: block;
	width: 300px;
	min-height: 260px;
	margin: 5% auto 0 auto;
	padding: 0;
	z-index: 99;
}
```

Now it's perfect and the report can be closed. 👍 

Thanks!

**Comment by Tangol** [comment]
Ref https://github.com/villagedefrance/OpenCart-Overclocked/commit/bab3a37ee827422e9a5a1718447f14fa6914ad35#commitcomment-27490405
